### ISSUE TITLE
Occur 'executing job failed: No such file or directory' error when open a Go file in Vim

### ISSUE THREAD

**Issue by hackerzgz** [josa42/coc-go#29]
I am newer for `coc.nvim` and I got the latest version `coc.nvim` and install `coc-go` using `:CocInstall coc-go`

But I got __executing job failed: No such file or directory__ error message after installed success.

When I use `:CocInfo` got the output below:

```
[Info  - 11:36:08 AM] go/packages.Load
	packages = 1

## Output channel: go

[Error  - 11:36:08 AM] Starting client failed: 
TypeError: Cannot read property 'reader' of undefined
    at /Users/HackerZ/.vim/plugged/coc.nvim/build/index.js:78637:48
```

I think something happened when `coc-go` attempt to install `gopls` or `gomodifytags` into `~/.config/coc/extensions/coc-go-data/bin`.

Look forward to your reply, thanks.

**Comment by Andilutten** [comment]
+1 , also experiencing the same issue

**Comment by lanej** [comment]
+1 Here is the output from `:messages`

```
[coc.nvim] Installed extension: coc-go                                                                                                                                                                                                 
[coc.nvim] error: UnhandledRejection: ENOENT: no such file or directory, mkdir '/home/jlane/.config/coc/extensions/coc-go-data/bin'                                                                                                    
Error: ENOENT: no such file or directory, mkdir '/home/jlane/.config/coc/extensions/coc-go-data/bin'                                                                                                                                   
    at Object.fs.mkdirSync (fs.js:885:18)                                                                                                                                                                                              
    at Promise (/home/jlane/.config/coc/extensions/node_modules/coc-go/lib/utils/config.js:19:26)                                                                                                                                      
    at new Promise (<anonymous>)                                                                                                                                                                                                       
    at Object.<anonymous> (/home/jlane/.config/coc/extensions/node_modules/coc-go/lib/utils/config.js:18:16)                                                                                                                           
    at Generator.next (<anonymous>)                                                                                                                                                                                                    
    at /home/jlane/.config/coc/extensions/node_modules/coc-go/node_modules/tslib/tslib.js:110:75                                                                                                                                       
    at new Promise (<anonymous>)                                                                                                                                                                                                       
    at Object.__awaiter (/home/jlane/.config/coc/extensions/node_modules/coc-go/node_modules/tslib/tslib.js:106:16)                                                                                                                    
    at Object.configDir (/home/jlane/.config/coc/extensions/node_modules/coc-go/lib/utils/config.js:12:20)                                                                                                                             
    at Object.<anonymous> (/home/jlane/.config/coc/extensions/node_modules/coc-go/lib/utils/tools.js:35:51)      
```

**Comment by alipiry** [comment]
> I am newer for `coc.nvim` and I got the latest version `coc.nvim` and install `coc-go` using `:CocInstall coc-go`
> 
> But I got **executing job failed: No such file or directory** error message after installed success.
> 
> When I use `:CocInfo` got the output below:
> 
> ```
> [Info  - 11:36:08 AM] go/packages.Load
> 	packages = 1
> 
> ## Output channel: go
> 
> [Error  - 11:36:08 AM] Starting client failed: 
> TypeError: Cannot read property 'reader' of undefined
>     at /Users/HackerZ/.vim/plugged/coc.nvim/build/index.js:78637:48
> ```
> 
> I think something happened when `coc-go` attempt to install `gopls` or `gomodifytags` into `~/.config/coc/extensions/coc-go-data/bin`.
> 
> Look forward to your reply, thanks.

Make sure the command `gopls` works in your command line, if it does not, set go in you `PATHZ

**Comment by josa42** [comment]
Should be fixed by #51
